### ISSUE TITLE
We need to get the company registered as a corporate secretarial firm

### ISSUE THREAD

**Issue by mengwong** [legalese-io/legalese-io.github.io#117]
I believe there is a course that leads to a test which one can pass. Please post details here.

**Comment by wabisabiyo** [comment]
**roadmap**
tl;dr:  to become a corporate secretarial firm, we need to get the _company_ registered as a filing agent. this is so that only _one_ of our employees needs to be certified as a _qualified individual_ in order to help our customers file details on ACRA. the rest can be unqualified as long as they have a singpass

So, our progression can be 2 tier:-
(1)  get qualified as a corporate secretarial firm (with at least one qualified individual in the company)
https://www.acra.gov.sg/Procedure_to_register_or_renew_registration_as_a_Filing_Agent.aspx
http://statutes.agc.gov.sg/aol/search/display/view.w3p;page=0;query=CompId:a7ce6e1e-2f3c-47fb-92f8-e41c7f0aa143;rec=0#pr7-he-.

(2) get qualified as chartered secretary (for public companies)
this is probably of less importance, as we have the validation and signalling mechanism recognised by the common man and investors under (1) above

**Comment by wabisabiyo** [comment]
Dear Alexis

Thank you for your email of 18 April.

To be qualified to register as a Qualified Individual (QI) with ACRA, one of the qualifications include being a member of the Singapore Association of the Institute of Chartered Secretaries and Administrators.
1.  To qualify for membership, you will first need to enrol in the International Qualifying Scheme (IQS), which is the examination scheme en route to membership.
2.  After you have passed all the examinations, you will be admitted to the Institute as a Graduate.
3.  With the relevant working experience, you would need to apply for the membership.

After you have elected to membership, you then would be qualified to register as a QI.

I hope I have answered your queries, feel free to contact me should you require any further clarification.

Yours sincerely
Tan Rui Ting (Ms)
Chartered Secretaries Institute of Singapore (CSIS)
(f.k.a SAICSA)
Membership Executive

---

alexis:
i satisfy the pre-requisites to enrol, already have the relevant work experience to qualify (subject to passing of the exams), and i probably can qualify for exemptions on some of the modules for the exam (i.e. lower costs)

http://www.saicsa.org.sg/english/studentservices/studentsreg.asp#

the next exams are in end nov. and the closing date for registration for that exam is 13 jun. shall i apply for this?

once i qualify, we can register the company as a filing agent with acra as we would have satisfied the requirement for at least one person being qualified as a QI.

**Comment by wabisabiyo** [comment]
On 17 June 2016 at 8:21:35 AM, acra_bizfile@acra.gov.sg (acra_bizfile@acra.gov.sg) wrote:

Dear Sir/Mdm,

Transaction Name: Application to Register as a Qualified Individual
Transaction No.: Q160018807

Your application for registration as a Qualified Individual has been approved on 17/06/2016.

Thank you.

Regards,
Accounting and Corporate Regulatory Authority (ACRA)
